statsmodels/stats/oneway.py

```diff
@@ -856,12 +1059,69 @@ def test_scale_oneway(data, method='bfm', center='median', transform='abs',
 
     Parameters
     ----------
-    data
-    method "equal", "unequal", "bfm"
+    data : tuple of array_like or DataFrame or Series
+        Data for k independent samples, with k >= 2.
+        The data can be provided as a tuple or list of arrays or in long
+        format with outcome observations in ``data`` and group membershipt in
+        ``groups``.
+    groups : ndarray or Series
+        If data is in long format, then groups is needed as indicator to which
+        group or sample and observations belongs.
+    method : {"unequal", "equal" or "bf"}
+        `use_var` specified how to treat heteroscedasticity, unequal variance,
+        across samples. Three approaches are available
+
+        "unequal" : Variances are not assumed to be equal across samples.
```

Comment:
You probably want * or - here.  These look like definition lists which will not render reasonablely.

Reply:
These also render well.